Cannot find module 'autocannon'

## Expected Behavior
<!--- If you're describing a bug, tell us what should happen -->
<!--- If you're suggesting a change/improvement, tell us how it should work -->

Did *not* pass `--autocannon`, should prompt:
```To generate the report press: Ctrl + C```
## Current Behavior
<!--- If describing a bug, tell us what happens instead of the expected behavior -->
<!--- If suggesting a change/improvement, explain the difference from current behavior -->

```
internal/modules/cjs/loader.js:582
    throw err;
    ^

Error: Cannot find module 'autocannon'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.resolve (internal/modules/cjs/helpers.js:30:19)
    at runTool (/Users/morgism/.nvm/versions/node/v10.14.0/lib/node_modules/clinic/bin.js:289:34)
```

## Steps to Reproduce (for bugs)
<!--- Provide an unambiguous set of steps to reproduce this bug -->
1. Install `2.1.0`
2. Run `clinic doctor -- node <script>`
3. Install `2.0.0` 
4. Works as expected


## Environment
<!--- Check with `clinic -v` and `clinic doctor -v` -->
<!--- e.g. Clinic v0.8.1 Doctor v2.4.1 -->
* Clinic version: `v2.1.0`

<!--- Check with `node -v` and `npm -v` -->
<!--- e.g. Node v10.2.1 npm v6.0.1 -->
* Node version: `v10.14.0`
* npm version:  `6.4.1`

<!--- See http://whatsmyos.com/ -->
<!--- e.g. OSX 10.11.5 -->
* Operating system and version: 
`macOS 10.12.6`

--- a/package.json
+++ b/package.json
@@ -23,6 +23,7 @@
     "@nearform/flame": "^3.4.1",
     "any-shell-escape": "^0.1.1",
     "async": "^2.6.0",
+    "autocannon": "^3.2.0",
     "commist": "^1.0.0",
     "cross-argv": "^1.0.0",
     "dargs": "^6.0.0",
@@ -40,7 +41,6 @@
     "update-notifier": "^2.5.0"
   },
   "devDependencies": {
-    "autocannon": "^3.1.0",
     "snazzy": "^8.0.0",
     "standard": "^12.0.0",
     "tap": "^12.1.0",

